ccloader/js/filemanager.js

```diff
@@ -216,6 +206,28 @@ export class Filemanager {
 		}
 	}
 
+	_receiveInfoFromAndroid() {
+		const extensionsProvider = window.CrossAndroidExtensionListProvider;
+		if (extensionsProvider && extensionsProvider.getExtensionListAsJson) {
+			this._androidExtensions = JSON.parse(extensionsProvider.getExtensionListAsJson());
+		}
+
+		const modsProvider = window.CrossAndroidModListProvider;
+		if (modsProvider && modsProvider.getModListAsJson) {
+			this._androidModFolders = [];
+			this._androidModPackages = [];
+			for (const entry of JSON.parse(modsProvider.getModListAsJson())) {
+				// The returned entries are filenames within assets/mods/. If the file
+				// is a directory, a slash is appended to it.

```

Comment:
This comment would be a bit easier to understand if it explained why the slash needs to be removed

Reply:
CCLoader fails in peculiar ways otherwise, e.g. asset overrides stop working.

Reply:
That's exactly what I want in the comment